src/util/inc/tqueue.h

```diff
@@ -47,6 +47,9 @@ void      *taosAllocateQitem(int size);
 void       taosFreeQitem(void *item);
 int        taosWriteQitem(taos_queue, int type, void *item);
 int        taosReadQitem(taos_queue, int *type, void **pitem);
+// special type search Qitem
+int        taosSearchQitem(taos_queue, int type, void **pitem);
```

Comment:
search in queue?

Reply:
yes , need remove Qitem from queue after async delete all rows from disk.